docs: own message customization example (#36)
pubnub/react-chat-components: lib/docs/stories/code-examples/customization.stories.mdx

import { Meta } from "@storybook/addon-docs/blocks";
import { MessageList, Chat } from "../../../src";
import { PubNubMock } from "../../../mock/pubnub-mock";
import { PubNubProvider } from "pubnub-react";
import users from "../../../../data/users.json";
import messages from "../../../../data/messages-lorem.json";
const pubnub = new PubNubMock({ uuid: "user_a56c20222c484ff8987ec9b69b0c8f5b" });
import "./customization.css";

<Meta title="Code Examples/Customization" />

# Customization examples

<br />
<br />

> ## Change style of current user messages

Reviewer: ```suggestion
## Customize messages sent by the current user
```

Differentiating the looks of messages, depending on who sent them, is a common use case across some

> types of chat.

Reviewer: ```suggestion
types of chat apps.
```

> React Chat Components do not come with a separate set of CSS variables for messages that were sent

Reviewer: ```suggestion
PubNub Chat Components for React do not come with a separate set of CSS variables for messages that were sent
```

> by the "current" chat user. A dedicated `pn-msg--own` CSS class should be used instead.

Reviewer: ```suggestion
by the current chat user. To change the default styling of such messages, use a dedicated `pn-msg--own` CSS class instead.
```
